frontend/packages/console-app/src/components/volume-snapshot/create-volume-snapshot/create-volume-snapshot.tsx

```diff
@@ -230,6 +219,7 @@ const CreateSnapshotForm = withHandlePromise<SnapshotResourceProps>((props) => {
   };
 
   const isBound = (pvc: PersistentVolumeClaimKind) => pvc?.status?.phase === 'Bound';
+  const kindId = `${VolumeSnapshotModel.apiGroup}~${VolumeSnapshotModel.apiVersion}~${VolumeSnapshotModel.kind}`;
```

Comment:
Use `referenceForModel`.

```suggestion
  const kindReference = referenceForModel(VolumeSnapshotModel);
```

Reply:
Cool, I wanted to ask, was pretty sure this had to exist :)